src/manifestv3.json

```diff
@@ -11,19 +12,21 @@
   "content_scripts": [
     {
       "matches": ["http://*/*", "https://*/*"],
-      "js": ["content/content.js"]
+      "js": ["content/content.js"],
+      "all_frames": true
     },
     {
       "run_at": "document_start",
       "matches": ["http://*/*", "https://*/*"],
-      "js": ["contentStatic/contentStatic.js"]
+      "js": ["contentStatic/contentStatic.js"],
```

Comment:
Not really important at the moment, but I find the `contentStatic` naming a little but confusing. Probably can be renamed to `polyfill`/`wm-polyfill`.

Reply:
Yeah, can do as follow-up. Will prefer to keep PR as small as possible.